src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensions.scala

```diff
@@ -265,6 +263,49 @@ class GoogleExtensions(val directoryDAO: DirectoryDAO, val accessPolicyDAO: Acce
     }
   }
 
+  def getArbitraryPetServiceAccountKey(user: WorkbenchUser): Future[String] = {
+    for {
+      pets <- directoryDAO.getAllPetServiceAccountsForUser(user.id)
+      key <- if(pets.nonEmpty) getPetServiceAccountKey(user, pets.head.id.project) else getDefaultServiceAccountForShellProject(user)
```

Comment:
Is there any reason why always returning `pets.head` could be problematic?  I suspect that this will usually return the same pet for the user, so could this cause any issues?  If this is OK, then perhaps it's a nit, but maybe we shouldn't call the method `getArbitrary...`, because it's not really arbitrary.

Reply:
I don't think it is a problem. It is arbitrary from the caller's perspective. The fact that it may be predictable is an implementation detail.